src/views/Hotels.vue

```diff
@@ -2,26 +2,24 @@
   <div>
     <!-- <NavigatorTop /> -->
     <!-- <MenuDesktop /> -->
-    <div class="lg:flex container mx-auto">
+    <div class="lg:flex container mx-auto px-2">
       <div class="w-12/12 lg:w-3/12 p-2 lg:p-0">
-        <PriceRangeSlider
-          :hotels="hotels"
-          :step="5"
-          v-on:priceReceive="(...price) => this.priceReceive(...price)"
-        />
-        <p>price range: {{ priceRange }}</p>
-        <HotelRatingFilter />
-        <TripAdvisorRating />
-        <HotelPropertyFilter />
-        <Facilities />
+        <div class="lg:hidden" @click="showFilter">show filter</div>
+        <div :class=" (filter == true ? '' : 'hidden lg:block' )">
```

Comment:
does it exist this syntax :D, plz check again does it work ??? 
i remember it be `:class="{!filter, 'hidden lg:block'}"`